app.json

```diff
@@ -3,6 +3,10 @@
   "env": {
     "USE_AUTH": "true",
     "USE_HELMET": "true",
+    "SYNC_STORYBLOK_ON_RELEASE": "false",
+    "REDIS_URL": {
+      "required": true
```

Comment:
Maybe we should spin up our own redis for each pr instance instead? So we don't mix the caches, i smell weird issues from using the same cache for multiple servers 😬

Reply:
+1

Reply:
Fixed 😬